### PR TITLE
PT-485 | Setup testcafe

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Add testcafe to project

- Add first testcafe test

### Closes
**[PT-485](https://helsinkisolutionoffice.atlassian.net/browse/PT-485):**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -5,9 +5,10 @@
     "plugin:prettier/recommended",
     "plugin:import/errors",
     "plugin:import/warnings",
-    "plugin:import/typescript"
+    "plugin:import/typescript",
+    "plugin:testcafe/recommended"
```
Comment (jorilindell) [inline .eslintrc.json:9]: You could also use https://github.com/testcafe-community/eslint-plugin-testcafe-community (the latest release is from current year), but I don't know does it make any difference.

**Review by jorilindell — APPROVED** [review]
Looks good.

File: .eslintrc.json (hunk 1 above)
Comment (Sartonon) [reply .eslintrc.json:9]: hmm seems to have the same rules but also some additional stuff

Comment (Sartonon) [reply .eslintrc.json:9]: "plugin:testcafe/recommended" is the one mentioned in the testcafe docs

**Pull request merged**